src/matlab2tikz.m

```diff
@@ -859,8 +859,9 @@ case guitypes()
 
     if ~isVisible(handle)
         % Setting hide{x,y} axis also hides the axis labels in Pgfplots whereas
-        % in MATLAB, they may still be visible. Well.
-        m2t = m2t_addAxisOption(m2t, 'hide axis');
+        % in MATLAB, they may still be visible. Instead use the following.
+        m2t = m2t_addAxisOption(m2t, 'axis line style={draw=none}');
```

Comment:
Please use:
```matlab
m2t = m2t_addAxisOption(m2t, 'axis line style', '{draw=none}');
m2t = m2t_addAxisOption(m2t, 'ticks', 'none');
```
instead, since those are easier to track (those make sure that `axis line style` and `ticks` are not being set elsewhere).